.gitignore

```diff
@@ -297,3 +297,53 @@ cython_debug/
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
+
+# Azure Functions stuff
+bin
+obj
+csx
+.vs
+edge
+Publish
+
+*.user
+*.suo
+*.cscfg
+*.Cache
+project.lock.json
+
+/packages
+/TestResults
+
+/tools/NuGet.exe
+/App_Data
+/secrets
+/data
+.secrets
+appsettings.json
+local.settings.json
+
+node_modules
+dist
```

Comment:
Please can we remove dist so that component folder commits its dist folder - you can add dist in a gitignore inside backend if needed

Reply:
Done, although I noticed `dist` is already in gitignore above anyway. I would think we don't want it in the repo because it can be generated from `src/` anyway right?